lighthouse-core/gather/gatherers/styles.js

```diff
@@ -129,13 +133,17 @@ class Styles extends Gatherer {
   afterPass(options) {
     return this.endStylesCollect(options.driver)
       .then(stylesheets => {
-        // Want unique stylesheets. Remove those with the same text content.
+        // Generally want unique stylesheets. Mark those with the same text content.
```

Comment:
Not sure where we actually needed to do this? It looks like the only other usage of `Styles` is the no flexbox which I would advocate warning even on multiple shadow DOM uses, etc